types/content-type-parser.d.ts

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
can we remove this line and make the anys become explicit types?

Reply:
Yes but there is this rule: https://github.com/fastify/fastify/blob/84187e553e7e1a3b2acf49e18472bb3feda82db6/types/.eslintrc.json#L27

Opening an issue to track this.